app/angular/src/client/preview/angular/utils.ts

```diff
@@ -0,0 +1,28 @@
+function getMeta(component, [name1, name2]: any, defaultValue) {
```

Comment:
Where can I put this util ? It's duplicated in two projects: `addons/knobs` and `app/angular`

Reply:
Hmm, @ndelangen, where would be the best place to put this util? 🤔

Reply:
I was under the impression that there was an ongoing effort to remove duplicate parts of the code to a core package. Dunno if that's still a thing for 3.3 tho. We might just leave this as is for the time being and wait for that core module to be ready (?)

Reply:
Yeah, I agree

Reply:
It was kinda decided to leave the refactor until after the initial release of 3.3, and release it as a patch. @shilman started the refactor branch, @Hypnosphi wanted to hold it off, if I remember correctly.

---

> Where can I put this util ? It's duplicated in two projects: `addons/knobs` and `app/angular`

Please give me a bit more context so I can answer your question